check_graphite.gemspec

```diff
@@ -20,6 +20,7 @@ Gem::Specification.new do |s|
 
   # specify any dependencies here; for example:
   # s.add_development_dependency "rspec"
+  s.add_runtime_dependency "linear-regression"
```

Comment:
Hm, philosophy. Define absolutely necessary. (If it added significant code, I would not have used the external module.) You are right however that the module needs to have its version fixed in the gemspec.